elements/pl-drawing/pl-drawing.py

```diff
@@ -1876,6 +1879,11 @@ def grade(element_html, data):
     def abserr(x, xapp):
         return np.abs(x - xapp)
 
+    def abserr_ang(ref, x):
+        ref = ref % 360
+        x = x % 360
```

Comment:
Do we need the two lines above?

Reply:
woops, you're totally right I don't think the mod is actually needed here